js/common.js

```diff
@@ -1166,9 +1173,13 @@ function relativeDate(str) {
                     : (tmp = Math.round(h)) === 1 ? __('an hour ago')
                         : d < 1.01 ? '%s hours ago'.replace('%s', tmp)
```

Comment:
Looks like all `%s xxx ago` strings are not translated here.

All occurences should be fixed like this, I guess:
```suggestion
                        : d < 1.01 ? __('%s hours ago').replace('%s', tmp)
```